drivers/scheduler/errors.go

```diff
@@ -75,6 +75,14 @@ type ErrFailedToValidateApp struct {
 	Cause string
 }
 
+// ErrFailedToUpdateApp error type for failing to update an app
+type ErrFailedToUpdateApp struct {
+	// App is the app whose validation failed
+	App *spec.AppSpec
+	// Cause is the underlying cause of the error
+	Cause string
+}
```

Comment:
Define the error method on this struct, else it will be not be an error type

Reply:
Added error method.